concrete/single_pages/dashboard/system/basics/timezone.php

```diff
@@ -86,12 +86,12 @@
             <label class="control-label">
                 <?php echo t('User-Specific Timezones') ?>
             </label>
-            <div class="checkbox">
+            <div class="form-check">
                 <label>
```

Comment:
No container <label>, follow bootstrap 4 classes with form-check-label.

Reply:
Done for the checkboxes but there seems to be a new problem that I didn't notice before.

The select2 dropdown for the Default Timezone selector is not styled at all. Is it supposed to be like that for now?